
class ResourceNotFound(Exception):
    pass


class DataValidationError(Exception):
    pass
